utils/build_swift/driver_arguments.py

```diff
@@ -0,0 +1,1295 @@
+               help='build the Debug variant of LLDB')
+        option('--lldb-build-variant', set_,
+               choices=build_variant_choices,
+               help='CMake build variant for LLDB')
+
+    with mutually_exclusive_group():
+        option('--debug-libdispatch',
+               set_('libdispatch_build_variant', const='Debug'),
+               help='build the Debug variant of libdispatch')
+        option('--libdispatch-build-variant', set_,
+               choices=build_variant_choices,
+               help='CMake build variant for libdispatch')
+
+    with mutually_exclusive_group():
+        option('--debug-libicu',
+               set_('libicu_build_variant', const='Debug'),
+               help='build the Debug variant of libdicu')
+        option('--libdicu-build-variant', set_,
+               choices=build_variant_choices,
+               help='CMake build variant for libicu')
+
+    with mutually_exclusive_group():
+        option('--debug-llvm',
+               set_('llvm_build_variant', const='Debug'),
+               help='build the Debug variant of LLVM')
+        option('--llvm-build-variant', set_,
+               choices=build_variant_choices,
+               help='CMake build variant for LLVM and Clang')
+
+    with mutually_exclusive_group():
+        option('--debug-swift',
+               set_('swift_build_variant', const='Debug'),
+               help='build the Debug variant of Swift')
+        option('--swift-build-variant', set_,
+               choices=build_variant_choices,
+               help='CMake build variant for Swift')
+
+    with mutually_exclusive_group():
+        option('--debug-swift-stdlib',
+               set_('swift_stdlib_build_variant', const='Debug'),
+               help='build the Debug variant of the Swift standard library')
+        option('--swift-stdlib-build-variant', set_,
+               choices=build_variant_choices,
+               help='CMake build variant for the Swift standard library')
+
+    # -------------------------------------------------------------------------
+    in_group('Host and cross-compilation build options')
+
+    option('--host-target', set_,
+           default=StdlibDeploymentTarget.host_target().name,
+           metavar='TARGET',
+           help='the host target. LLVM, Clang, and Swift will be built for '
+                'this target. The built LLVM and Clang will be used to '
+                'compile Swift for the cross-compilation targets.')
+
+    option('--cross-compile-hosts', append(separator=' '),
+           metavar='TARGET',
+           help='space-separated list of targets to cross-compile the host '
+                'Swift tools for. Can be used multiple times.')
+
+    option('--stdlib-deployment-targets', append(separator=' '),
+           metavar='TARGET',
+           help='space-separated ist of targets to compile or cross-compile '
+                'the Swift standard library for. %(default)s by default')
```

Comment:
Whats up with the format string here?

Reply:
The magical `%(default)s` format string inputs the `default` value when the argument parser outputs the help message.